sdk/metric/meter_test.go

```diff
@@ -960,9 +960,6 @@ func TestGlobalInstRegisterCallback(t *testing.T) {
 	_, err = preMtr.RegisterCallback(cb, preInt64Ctr, preFloat64Ctr, postInt64Ctr, postFloat64Ctr)
 	assert.NoError(t, err)
 
-	_, err = preMtr.RegisterCallback(cb, preInt64Ctr, preFloat64Ctr, postInt64Ctr, postFloat64Ctr)
-	assert.NoError(t, err)
-
```

Comment:
Is this related to the changes, or just clean up?

I'm fine either way, just wondering if I understand this.

Reply:
Just cleanup, but revealed by the change.  The callback was registered twice (by accident?), which now causes values to be 2x.